debian/assets/carlos_ctl/cli.py

```
@@ -0,0 +1,204 @@
+  /etc/carlos-emr/carlos-emr.env       site settings -> carlos-ctl init-config
+  /etc/carlos-emr/carlos.properties    app config    -> carlos-ctl restart
+  /etc/carlos-emr/backup.env           backups       -> next timer run; prove
+                                                        with carlos-ctl backup full
+  /etc/carlos-emr/modsecurity/         WAF policy    -> carlos-ctl waf reload
+  /etc/carlos-emr/tomcat/              Tomcat        -> carlos-ctl restart
+
+Full documentation: /usr/share/doc/carlos-emr/README.Debian
+"""
+
+
+def _cmd_status(argv) -> int:
```

Ruff (0.16.1)

[warning] Unused function argument: `argv`

(ARG001)